src/SchemaReader.php

```diff
@@ -236,6 +238,24 @@ private function fillAttribute(AttributeSingle $attribute, \DOMElement $node): v
         if ($node->hasAttribute('use')) {
             $attribute->setUse($node->getAttribute('use'));
         }
+
+        $attribute->setMeta($this->loadMetaAttributesForElement($attribute, $node));
+    }
+
+    private function loadMetaAttributesForElement(SchemaItem $item, \DOMElement $node): array
```

Comment:
Wonder whether this should also be done for elements in the same step.

For example this would include dfdl attributes from something like `<xsd:element dfdl:encoding="iso-8859-1" dfdl:initiator="UNA" dfdl:length="6" dfdl:lengthKind="explicit" dfdl:terminator="%NL;%WSP*; %WSP*;" minOccurs="0" name="UNA" type="srv:UNA"/>`

Hence, it could be interesting to move the meta data array into a `MetaDataBagTrait`...

WDYT?

Reply:
Thanks for the review @Guite.
Parsing the metadata from the element is simple enough.
I've added it to this PR.